### DISCUSSION TITLE
DatePicker limit range

### DISCUSSION THREAD

**Discussion by PavloParafiloCricut** [Megabit/Blazorise#4371] [Q&A]
Please advise how to validate DatePicker  Range 
EX: only 1 year span should be selected

**Comment by stsrki** [comment]
What type of validation do you use? Annotations, handlers, ?

**Reply by PavloParafiloCricut** [reply]
Hm. Anything so far. It would be great to have some param like MaxRange in component. But any other solution is fine too

**Reply by stsrki** [reply]
I think you will need to create new custom validation for the date range field. Look at https://learn.microsoft.com/en-us/previous-versions/aspnet/cc668224(v=vs.100) for reference.

**Comment by PavloParafiloCricut** [comment]
@stsrki one more question please.
I see that date-picker has <html>
<body>


Step | The step attribute specifies the legal day intervals to choose from when the user opens the calendar in a date field.
-- | --



</body>
</html>

It looks like something that may work for my "validation". But I don't see this parameter. "Blazorise" Version="1.1.4.1"

https://blazorise.com/docs/components/date-picker

**Reply by stsrki** [reply]
Hm, I think it accidentally went into the DatePicker documentation. Copy/pasta from DateEdit page 😅

**Reply by PavloParafiloCricut** [reply]
It's a pity. Any plans to implement it? :)

**Reply by stsrki** [reply]
DatePicker is based on https://flatpickr.js.org/options/, and it doesn't have a step option. They only have `hourIncrement` and `minuteIncrement`.
